osgp/shared/osgp-ws-publiclighting/src/main/resources/schemas/pl-adhocmanagement.xsd

```diff
@@ -287,6 +259,46 @@
     </xsd:sequence>
   </xsd:complexType>
 
+  <xsd:complexType name="Status" />
```

Comment:
If you change this to `<xsd:complexType name="Status" abstract="true" />` the generated class for `Status` will be abstract, which appears to be a better fit with the interface for the related domain objects.